README.md

```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
 ### Required for Windows:
 * `python` installed
 * `cmake`
-* Visual Studio compilers, version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+* Visual Studio with release year not older than 2015 and Visual Studio compiler, which version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+
 
 
 ### Before building do:
 * On Linux:
   * `git submodule update --init --recursive`
   * `cd 3rd_party && ./prereq-build.sh && cd ..`
 * On Windows:
-  * `git submodule update --init --recursive`
-  * `cd 3rd_party/protobuf-c && git apply ../protobuf-c-vs2008-support.patch && cd ../..`
+  * Update third-party intel-xed, mbuild, protobuf, protobuf-c and safestringlib submodules (3rd_party folder):
```

Comment:
```suggestion
  * Update third-party intel-xed, mbuild, protobuf, protobuf-c and safestringlib submodules (`3rd_party` folder):
```

Reply:
It seems you didn't apply my suggestion here, why?

Reply:
Suggestion was rejected by mistake, now quotes are added.